src/JustEat.StatsD/IpTransport.cs

```diff
@@ -30,6 +31,20 @@ public void Send(string metric)
             }
         }
 
+        public void Send(ArraySegment<byte> metric)
+        {
+            if (metric.Count == 0)
```

Comment:
Can `ArraySegment<>` be `null`, or is it a `struct`?

Reply:
but array needs to be checked